DQM/Integration/python/clients/beamhlt_dqm_sourceclient-live_cfg.py

```diff
@@ -134,7 +134,7 @@
 
 # Set rawDataRepacker (HI and live) or hltFEDSelectorTCDS+hltFEDSelectorOnlineMetaData (for all the rest)
 if (process.runType.getRunType() == process.runType.hi_run and live):
-    rawDataInputTag = "rawDataRepacker"
+    rawDataInputTag = "hltFEDSelectorTCDS"
```

Comment:
Now the two clauses are identical:

https://github.com/cms-sw/cmssw/blob/cea5e6fd5b1b8d8f3fd6deae14dbfe9d87d718db/DQM/Integration/python/clients/beamhlt_dqm_sourceclient-live_cfg.py#L136-L143

why keeping the `else/if` structure?

Reply:
I agree with you Marco! This was just dictated by the urgency of providing a PR to fix the online clients in data-taking! I can code it better once the HI frenzy is over!
Btw this is highly dependent on the menu, so we should clarify with HLT and HIN experts what will be done in the next years for HI data-taking.

Reply:
Addressed in [0ce5001](https://github.com/cms-sw/cmssw/pull/40109/commits/0ce500163e67b1d7270b0944f8bbd5563928139d)